double:
  acceleration_covariance: 0.0001
  gyroscope_covariance: 0.0001
  integration_covariance: 0.0001
  bias_covariance: 0.001
  use_second_order: 0.0
  origin_x: 0.0
  origin_y: 0.0
  gravity: 9.81
  bias_num_measurements: 200.0
  gps_noise: 2.5
  odom_noise: 0.1
  heading_noise: 0.1 #radians
